nixos/modules/services/networking/xrdp.nix

```diff
@@ -26,6 +26,9 @@ let
     substituteInPlace $out/sesman.ini \
       --replace LogFile=xrdp-sesman.log LogFile=/dev/null \
       --replace EnableSyslog=1 EnableSyslog=0
+
+    # Ensure that clipboard works for non-ASCII characters
+    sed -i -e '/.*SessionVariables.*/ a\'$'\n'"LANG=${config.i18n.defaultLocale}"'$\n'"LOCALE_ARCHIVE=${config.i18n.glibcLocales}/lib/locale/locale-archive;" $out/sesman.ini
```

Comment:
Can you break this up? It's difficult to read.

Reply:
Done